standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -144,6 +144,8 @@
   private static String[] processorCapabilities;
   private static String processorIdentifier;
 
+  private ValidTxnWriteIdList txnWriteIdList;
+
```

Comment:
I am not comfortable having it here :(
We should push the value through request, and do not store state in the HMS client